dispatcher/nm-dispatcher.c

```diff
@@ -627,6 +619,26 @@ find_scripts (const char *str_action)
 	}
 	g_dir_close (dir);
 
+out:
+	return sorted;
```

Comment:
above `g_slist_insert_sorted` is wrong. Now that you the files are in different paths, you need to sort based on the base-name.

A simple fix would be to implement a compare function that looks into the full path and compares the base-names.

Instead of incrementally constructing the list, it seems better to just construct it via `g_slist_prepend()` and do one final g_slist_sort().

Reply:
Okay. Fixing...